mono/metadata/abi-details.h

```diff
@@ -23,29 +23,33 @@
  * is given by the --with-cross-offsets= configure argument.
  */
 
```

Comment:
Why is this needed ?

Reply:
Because I decided I don't want to add an offsets file for wasm. We discussed a thousand times about getting rid of those annoying thing and start to support a retargetable runtime.

This PR includes the beginning of that. It only does for alignment info and it only has info for wasm. I hope we eventually include all our targets in abi.c (or whatever we decide to use instead) and the offset files stop being a thing.

Reply:
Alignment determines offsets/sizes, so hardcoding just the offsets will not work.

Reply:
I'm fully aware of it.
This is what I needed so far, the rest will come later as part of the wasm AOT work.